completions/git.fish

```diff
@@ -1,44 +1,3 @@
-#!fish
-#
-# git-flow-completion
-# ===================
-#
-# Fish completion support for [git-flow](http://github.com/nvie/gitflow)
-#
-# The contained completion routines provide support for completing:
-#
-#  * git-flow init and version
-#  * feature, hotfix and release branches
-#  * remote feature, hotfix and release branch names
-#
-#
-# Installation
-# ------------
-#
-# To achieve git-flow completion nirvana:
-#
-#  1. Install this file in your `~/.config/fish/completions` folder.
-#
-#
-# The Fine Print
-# --------------
-#
-# Copyright (c) 2012-2015 [Justin Hileman](http://justinhileman.com)
-#
-# Distributed under the [MIT License](http://creativecommons.org/licenses/MIT/)
```

Comment:
leave the header.

this file is literally copied from another repo. don't mess things up by removing copyright headers :)